docs/UserGuide.md

```diff
@@ -380,13 +381,37 @@ Format: `sort [sna/] [snd/] [ssa/] [ssd/] [so/]…​`
 
 ### Importing a show: `import`
 
+**Description:** Want to easily import Trackermon data from other devices? Just use our import function!
+
+**Format:** `import`
+
+**Example & Output:** `import`
+- Step 1. Navigate to the Trackermon data you want to import using your OS' native GUI!
```

Comment:
Maybe you could link GUI to the glossary? 🤔

Reply:
Nice catch! I'll update the UG.